code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -129,6 +129,7 @@
 			volume = affected.reaction_results["fire"]*FIRE_GROWTH_RATE
 			location.assume_air(affected)
 
+	location.PolluteListTurf(list(/datum/pollutant/smoke = 15, /datum/pollutant/carbon_air_pollution = 5), POLLUTION_ACTIVE_EMITTER_CAP)
```

Comment:
The way it works on horizon is that hotspots don't make any pollution, but they can create(30%) a residual turf fire when they dissipate, which then emitts pollution in a more controlled and slow way (as highlighted here https://user-images.githubusercontent.com/6381979/126042078-cd21a363-2c08-4e2c-8bcd-f36e8dfd53fc.png the entire room was on fire). What's done here is a really excessive emission, I'd suggest changing that